libbeat/common/backoff/backoff.go

```diff
@@ -17,10 +17,13 @@
 
 package backoff
 
+import "time"
+
 // Backoff defines the interface for backoff strategies.
 type Backoff interface {
 	Wait() bool
 	Reset()
+	WaitDuration() time.Duration
```

Comment:
I like this interface workaround for the new setting! Please comment the new function to clarify that `Wait` and `WaitDuration` should have the same effect on the target's internal state, i.e. that repeatedly calling `<-time.After(b.WaitDuration())` is ~equivalent to repeatedly calling `b.Wait()`, to be used when the wait needs to be handled manually.

Reply:
Added in bb766b512220bc4bf0c1a422d6bb311ea186e358.